MapboxNavigation/RouteMapViewController.swift

```diff
@@ -47,8 +47,6 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
     var arrowCurrentStep: RouteStep?
     var isInOverviewMode = false
 
-    let overviewContentInset = UIEdgeInsets(top: 65, left: 15, bottom: 55, right: 15)
```

Comment:
Overview mode required some extra padding on all sides to account for the marker height. How does it look right now?